sql/maintenance_utils.sql

```diff
@@ -42,3 +34,17 @@ CREATE OR REPLACE FUNCTION recompress_chunk(
     if_not_compressed BOOLEAN = false
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_recompress_chunk' LANGUAGE C STRICT VOLATILE;
 
+-- Experimental features
+CREATE OR REPLACE PROCEDURE timescaledb_experimental.move_chunk(
```

Comment:
I suggest putting these functions in `ddl_experimental.sql` so that we collect all experimental DDL commands in one file.